plugins/helphub-contributors/.gitignore

```diff
@@ -0,0 +1,12 @@
+# OS junk and project's build
+node_modules
+.directory
+.DS_Store
+Desktop.ini
+.Trash-*
+
+# third party assets
+admin/assets/
+
+# generated assets
+public/css/helphub-contributors-public.css
```

Comment:
There shouldn't be a need to add this file, any required files or directories to be ignored should be added in the main project `.gitignore` located at https://github.com/Kenshino/HelpHub/blob/master/.gitignore